### PR TITLE
About Page?

### PR DESCRIPTION
**Pull request by benhamill**
Adds an About page so people know who's behind this site (and can infer our biases).

### CODE REVIEW

**Comment by benhamill** [comment]
I'm not sure where to link this from. I thought maybe the footer, but IDK. Also I notice the footer says it's all CC-BY @h3h. Which I'm happy with, but thought I'd bring it up in case it should be changed.

File: about.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Austin Propositions is a site written and maintained by a small group of
+Austinites who got tired of being confused by ballot language of propositions
+that go to the whole city in elections. They decided to do research on each one
+and share what they found with anyone who would find it useful.
+
+The current authors are:
+* [Brad Fults](https://bradfults.com)
+* [Ben Hamill](http://benhamill.com)
+* [James Martin](#)
+* [Jimmy Lyke](#)
```
Comment (benhamill) [inline about.md]: @jameslmartin @jkyle: What would y'all wanna link to? I feel like "Facebook" or even "Twitter" would be suboptimal, but I'm willing to accept the idea that I'm a fringe radical in that respect. 😜

Comment (benhamill) [inline about.md]: @jameslmartin @jlyke: What would y'all wanna link to? I feel like "Facebook" or even "Twitter" would be suboptimal, but I'm willing to accept the idea that I'm a fringe radical in that respect. 😜

Comment (benhamill) [reply about.md]: OTOH, we could write a little 2-sentence bio about each of us and link or not link from there individually. IDK which is better.

**Comment by h3h** [comment]
I like the idea of each contributor adding their own About page line when they submit their first PR. That way it's tied to real contribution, not the best of plans.

**Comment by h3h** [comment]
Also needs a rebase on `master`.

**Comment by benhamill** [comment]
Rebased. But still not sure where to link it from.

**Comment by h3h** [comment]
I’ll add a sub header or something.

**Comment by benhamill** [comment]
What about this? I did a footer thing.

File: _includes/footer.html [hunk 2]
```diff
@@ -1,4 +1,9 @@
-<p class="site-footer">
+<div class="site-footer">
+  <ul>
+    <li><a href="/about">About</a></li>
```
Comment (benhamill) [inline _includes/footer.html]: To look perfect, these all have to be on the same line, which is kinda butts, but… 🤷‍♂️

**Pull request merged**